.github/workflows/fineract-build-push.yml

```diff
@@ -0,0 +1,136 @@
+name: Publish Packages and Images
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - staging
+      - dev
+      - build-push/VL-1536
```

Comment:
Don't forget to remove `build-push/VL-1536` once you are done testing the pipeline